### DISCUSSION TITLE
Tidligere gruppevagter forsvinder ikke

### DISCUSSION THREAD

**Discussion by MartinQwist** [xooom-community/public#39] [Bugs] [answered]
Tidsligere vagter fremgår stadig af vagtoversigten, hvis vagterne er lagt ind i en gruppe. De vises ikke på portalen. 

Se eksempel nedenfor for d. 25. og 30. april, som stadig fremgår af listen. Her er tilmelding luk netop ændret, men indtil da kunne man faktisk stadig melde sig på ventelisten til et tidligere event. 

Vagter, der er eksklusive for grupper, skal gerne forsvinde fra vagtoversigten, når de er overstået. Ligesom de gør på portalen. Ventelisten skal bør desuden lukkes for tidligere vagter. 

![image](https://github.com/xooom-community/product/assets/167326849/ebb09ba2-563e-4ed5-9e26-2f8c10a959c5)

**Comment by mikkelsl** [comment, marked as answer]
Løses her: https://github.com/orgs/xooom-community/projects/1/views/1?pane=issue&itemId=62136971
